hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/repository/ContractActionRepositoryTest.java

```diff
@@ -0,0 +1,24 @@
+package com.hedera.mirror.web3.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.hedera.mirror.common.domain.contract.ContractAction;
+import com.hedera.mirror.web3.Web3IntegrationTest;
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.Test;
+
+@RequiredArgsConstructor
+public class ContractActionRepositoryTest extends Web3IntegrationTest {
+    private final ContractActionRepository contractActionRepository;
+
+    @Test
+    void findAllByConsensusTimestampSuccessful() {
+        var timestamp = domainBuilder.timestamp();
+        ContractAction contractAction = domainBuilder.contractAction()
+                .customize(c -> c.consensusTimestamp(timestamp))
+                .persist();
+        assertThat(contractActionRepository.findAllByConsensusTimestamp(timestamp))
```

Comment:
normally we just do

```suggestion
        var contractAction = domainBuilder.contractAction().persist();
        assertThat(contractActionRepository.findAllByConsensusTimestamp(contractAction.getConsensusTimestamp()))
```

can keep it as is